### ISSUE TITLE
Setup not working

### ISSUE THREAD

**Issue by userless256** [intel/openvino-plugins-ai-audacity#128]
The v3.5 installer doesn't work. I always get an error message when setup starts to download additional files saying "Error sending data (12175) A security error has occurred". I've tried to disable any firewall or antivirus, also updated my system but it doesn't work. I'm not sure if it's just my machine out of date (wich I don't think so because previous release worked perfectly), or a general problem.

**Comment by RyanMetcalfeInt8** [comment]
Hi @userless256,

Hmm, strange. Can you send a screenshot?

Thanks,
Ryan

**Comment by RyanMetcalfeInt8** [comment]
And what version of Windows are you using?

**Comment by petersampsonaudacity** [comment]
Installed fine for me on W10 (64-bit)

**Comment by 3point141597** [comment]
also just got 
"Error sending data (12175) A security error has occurred"
![OpenVINO AI Plugin error](https://github.com/intel/openvino-plugins-ai-audacity/assets/168042015/d9fc0652-7d54-473a-a3b1-85a9517a4c4b)

**Comment by 3point141597** [comment]
"Error sending data (12175) A security error has occurred"

Version
![Version Audacity 3 5 1](https://github.com/intel/openvino-plugins-ai-audacity/assets/168042015/9b85a78b-1e94-4662-a44d-8b7cc36b58b4)

64 bit
![Version Audacity 64 bit](https://github.com/intel/openvino-plugins-ai-audacity/assets/168042015/183696f0-c658-41ad-836b-078ce410b94f)

**Comment by userless256** [comment]
Yes. It seems like Win 8/8.1 (my version) has problems with the setup.

**Comment by userless256** [comment]
> also just got "Error sending data (12175) A security error has occurred" ![OpenVINO AI Plugin error](https://private-user-images.githubusercontent.com/168042015/325416943-d9fc0652-7d54-473a-a3b1-85a9517a4c4b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQwMTI4NTIsIm5iZiI6MTcxNDAxMjU1MiwicGF0aCI6Ii8xNjgwNDIwMTUvMzI1NDE2OTQzLWQ5ZmMwNjUyLTdkNTQtNDczYS1hM2IxLTg1YTk1MTdhNGM0Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyNVQwMjM1NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01ZWRjMGZjOTg1ODA0NzE0MmEyYWU5ZjgzYTQ5ZjFkZGJiMDFmZDkxYzUwNDhkOTkxZmI2NjA0NjQ3MTZjMDY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.K6n7TZ9QuCKloScWZbPBAH9_XZd69f4KurpyVjhTZrI)

My issue looks like that too

**Comment by 3point141597** [comment]
Also have Windows 8.1
How embarrassing!

**Comment by RyanMetcalfeInt8** [comment]
Hi all, just wanted to check in and say that, unfortunately, I don't have any good leads about what the issue could be. There have been a number of reports of this, but it's not my area of expertise... I'll tag this as a 'good first issue', in case someone from the community has an idea about what it could be, and what potential workarounds there are...

**Comment by RyanMetcalfeInt8** [comment]
Hi all,

I saw in issue #164 that running the installer as administrator seemed to help. Has anyone else tried that?

Thanks!
Ryan

**Comment by Nyscorp** [comment]
Im getting the same thing but with a different error code, I have all my fire walls off trying to get his to work
![Screenshot (143)](https://github.com/intel/openvino-plugins-ai-audacity/assets/146999203/8f5d6be4-e078-4e9f-8613-003bba45ed6c)

**Comment by userless256** [comment]
Tried to run the lastest installer as administrator. Same error code
![325416943-d9fc0652-7d54-473a-a3b1-85a9517a4c4b](https://github.com/intel/openvino-plugins-ai-audacity/assets/108380490/1eba6e51-198b-422c-84e3-12aa5efbffd3)

**Comment by RyanMetcalfeInt8** [comment]
Hi all, can you please report which version of Windows you are using?

**Comment by petersampsonaudacity** [comment]
![image](https://github.com/intel/openvino-plugins-ai-audacity/assets/50205984/cad25839-d37f-45cd-b39e-7b49063f3c1b)

**Comment by Nyscorp** [comment]
![Screenshot (146)](https://github.com/intel/openvino-plugins-ai-audacity/assets/146999203/8260cd2e-3a77-4860-84bd-edffcacbbc66)

**Comment by Nyscorp** [comment]
it just worked with that running it not as admin and with all normal firewalls

**Comment by RyanMetcalfeInt8** [comment]
> it just worked with that running it not as admin and with all normal firewalls

Got it -- so just to confirm, it's working for you now? Any idea what changed?

**Comment by Nyscorp** [comment]
I don't think so but my computer might have updated overnight, i know it at least restarted it's self overnight

**Comment by RyanMetcalfeInt8** [comment]
> Yes it worked and is installed. I don't think so but my computer might have updated overnight, i know it at least restarted it's self overnight

Got it. I do see that the 'OS build' listed there is only a couple days ago.. so maybe that fixed it?

**Comment by endricowennaldy** [comment]
got the same issue, tried running the installer as admin but nothing changed
![Capture](https://github.com/intel/openvino-plugins-ai-audacity/assets/47880166/6fb1aaf0-885e-4fac-b7a4-0c4fa65a70f9)

**Comment by h15961** [comment]
Some countries and regions are unable to connect to the huggingface website, and installing version 3.5.1 of the plugin cannot download and install the model.
The installation of plugins in version 3.4.2 does not have these issues.

**Comment by RyanMetcalfeInt8** [comment]
> Some countries and regions are unable to connect to the huggingface website, and installing version 3.5.1 of the plugin cannot download and install the model. The installation of plugins in version 3.4.2 does not have these issues.

Hmm, this is a tricky one as huggingface is where the models are kept. Yes, the 3.4.2 release posted the model zip files to github.com directly, but this approach wasn't feasible as the number of models increased.

**Comment by Metaverse280** [comment]
My installer is also giving the same error, could you provide the zip version of the latest version of OpenVino AI effects?

**Comment by crisc2000** [comment]
The same error for me in v3.5.1-R2.2 . I use Windows 7 x64. I have tried to enable TLS 1.2 and 1.3, but no change

**Comment by ExGuestCJ** [comment]
Error happens in latest version aswell

**Comment by grzkwt** [comment]
I switched my VPN to Germany and issue gave away ;)
Looks like server(s) are unavailable for some regions.

**Comment by ExGuestCJ** [comment]
Has this ever been fixed?

**Comment by grzkwt** [comment]
Just successfully upgraded OpenVINO AI Plugins from v3.6.1-R3.1 to v3.6.2-R3.2 without any issues and VPN's "black magic".

**Comment by ExGuestCJ** [comment]
I'll take that as a yes

**Comment by RyanMetcalfeInt8** [comment]
This is a tricky one. It's hard to say whether it is 'fixed' for everyone, as it relies on connectivity to hugging face servers which appear to be sensitive to various regions..

**Comment by ExGuestCJ** [comment]
Ok so I tried again, without admin privileges, with admin privileges, and with the antivirus disabled and I still get the error

**Comment by ExGuestCJ** [comment]
Still happens in v3.6.4 R3.4 executable

**Comment by ExGuestCJ** [comment]
> Still happens in v3.6.4 R3.4 executable

The weird part is that the huggingface site works perfectly here in venezuela

**Comment by venomNewtona** [comment]
Hey guys,

I have two PCs:  Win-7 and Win-10, my installation report is below, just FYI...

**1.** 
I keep getting the "security issue" when installing 
audacity-win-v3.6.4-R3.4-64bit-OpenVINO-AI-Plugins
over previously installed
audacity-win-3.6.4-64bit
on my Windows-7 PC 

But the installation goes fine if I uncheck everything, I guess only "core files" installed.

**2.**
On my second Windows-10 PC the installation passed w/o problems, and the needed functionality works perfectly (speech-to-text).

**3.** 
Then I copy the "openvino-models" folder contents (speech recognition files) from Win10 PC to Win7 (with incomplete installation).
I thought this folder is the only difference, and expected my trick works but it doesn't  :-(
I reinstalled everything a few times, no success. 

It's always goes the same way under my Win-7:
- I turn on the module in preferences
- restart Audacity
- I'm instantly getting the error like   "unsuitable module",  "cannot load the module mod-openvino" and "error: the specified procedure was not found"  — all in one error pop-up, pressing Ok results in a normal Audacity start, but with inactive openvino module.
![nepodhod module](https://github.com/user-attachments/assets/95fbe9dc-34e3-426f-831d-2864c099d5fc)

I'm happy with the result and I'm posting this just to shed more light on the problem.

**Comment by Etchegando59** [comment]
Alguém conseguiu resolver este problema ?
